sdk/core/azure-core/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 ## 1.9.1 (Unreleased)
```

Comment:
Given this is a new feature, it should be 1.10.0

Reply:
fixed.